neqo-qpack/src/decoder.rs

```diff
@@ -501,7 +504,9 @@ impl QPackDecoder {
         if ack_increment_delta > 0 {
             self.send_buf
                 .encode_prefixed_encoded_int(0x00, 2, ack_increment_delta);
-            self.table.increment_acked(ack_increment_delta);
+            if self.table.increment_acked(ack_increment_delta).is_err() {
+                panic!("This should never happen");
+            }
```

Comment:
Use `x.unwrap("This should never happen")` instead of `if x.is.err() {panic!()}`